src/saved_game.hpp

```diff
@@ -143,6 +147,8 @@ class saved_game
 	config starting_pos_;
 
 	replay_recorder_base replay_data_;
+
+	bool skip_story_;
```

Comment:
You need to initialize this boolean, otherwise its initial value is indeterminate.

Please also copy its value in the copy constructor.

Reply:
Fixed. Thanks.